src/main/webapp/index.html

```diff
@@ -16,6 +17,9 @@
       <div id="form-title">
         <p>Copy your Tweet here</p>
       </div>
+      <label><input type="checkbox" onclick="setLocationCookie()" /> Use Location</label>
```

Comment:
[Optional] Is best practice to keep javascript separate from markup, so I would remove the onclick attribute and instead add a event listener to javascript, or even better use jQuery.

Reply:
Which javascript file would this be in?

Reply:
index.js. If it doesn't exist, create it and add the function.